controller/user.js

```diff
@@ -171,29 +171,31 @@ async function verifyAccount(req, res) {
   });
 }
 
-async function update(req, res, next) {
-  const id = getIdParam(req);
-  if (!req.body.id) {
-    throw new errors.FcError(errors.MISSING_USER_ID);
-  } else {
-    models.User.update(req.body, {
-      where: { id: id },
-    })
-      .then((rowsUpdate) => {
-        if (rowsUpdate[0] == 0) {
-          throw new errors.FcError(errors.USER_NOT_FOUND);
-        } else {
-          models.User.findByPk(id).then((user) => {
-            res.status(statusCode.SUCCESS).json({
-              code: 0,
-              msg: "",
-              data: { user: buildUserJson(user) },
-            });
-          });
-        }
-      })
-      .catch(next);
+async function update(req, res) {
+  const user_id = !req.params.id ? req.user.id : getIdParam(req);
```

Comment:
Update mình không cho truyền id nha. 
Vì hiện tại user mình chưa có role. Mình ko cho phép update information của một user khác.

Reply:
Oh but that's ok vì chỉ mỗi api /me gọi nó.